### ISSUE TITLE
Teleport (Time travel)

### ISSUE THREAD

**Issue by Pagec531** [PlaceholderGames/2016-Team2Thirds#7]
Allow the escaping player to teleport to 3 different areas. These will be the 3 different maps that will make it seem as if they have traveled forward and back in time.

**Comment by Pagec531** [comment]
Got a working teleport for the upper Z axis.

Update\* Just fixed it so that the player will teleport up from his current location rather than to a given location every time.

Currently working so that the player may teleport down.

**Comment by Pagec531** [comment]
I have uploaded the new branch could you guys download it to test it for any bugs thanks. Once tested and complete i can merge.

**Comment by KieranClarkee** [comment]
I have done a bit of testing of the teleport system. It's done well. The only thing I would suggest is that you give a little a little bit of space when you teleport so the player is slightly off the ground when going to the different level as it would give us a tiny margin for error when creating the maps themselves.

**Comment by Pagec531** [comment]
Thanks for the feedback I agree on that and I will edit it now along with adding the second teleportation. Is the keybindings okay for it as well F for up G for down? Maybe X for down you have the spacing on the keys then that match up and down future and past?

**Comment by Pagec531** [comment]
Update - I have now added the ability to teleport down using the X key along with these changes I have added two platforms one above the starting position and below just for testing purposes.

**Comment by Pagec531** [comment]
Update- I have checked over the spacing between teleporting and landing and that's ok now there is some space when landing. Also comments have been added on the blueprints explaining which ones which. Once we get to actually creating the level and designing it we will have to look back at teleporting to make sure the distance between each level matches. I will open up an updated issue for this when it comes to that.
